hugo/data/teams.yml

```diff
@@ -21,7 +21,7 @@ sections:
         members:
         - picture: santiago.png
           name: Santiago Mola
-          position: Head of architecture
+          position: VP of Engineering
```

Comment:
Santiago should be in the Management team now.

Reply:
Thanks @marnovo! Done.